app/lib/submission_builder/ty2024/states/md/documents/md502.rb

```diff
@@ -86,25 +86,27 @@ def document
               add_element_if_present(xml, "Blind", :MD502_LINE_B_SPOUSE_BLIND)
             end
           end
-          xml.Standard do
-            xml.Count calculated_fields.fetch(:MD502_LINE_A_COUNT)
-            xml.Amount calculated_fields.fetch(:MD502_LINE_A_AMOUNT)
-          end
+        end
+        xml.Standard do
```

Comment:
Do we want to include this if there are no values present?

Reply:
Yes, see Jira conversation 

https://codeforamerica.atlassian.net/browse/FYST-1717